prow/plugins/lifecycle/reopen.go

```diff
@@ -63,37 +63,39 @@ func handleReopen(gc githubClient, log *logrus.Entry, e *github.GenericCommentEv
 		return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, commentAuthor, response))
 	}
 
-	// Add a comment before reopening the PR or issue
-	// to leave an audit trail of who asked to reopen it.
 	if e.IsPR {
-		response := fmt.Sprintf("Reopening this PR.")
-		if err := gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, commentAuthor, response)); err != nil {
-			log.WithError(err).Errorf("Failed adding comment while reopening the PR")
-		}
-
 		log.Infof("/reopen PR")
 		err := gc.ReopenPR(org, repo, number)
 		if err != nil {
 			if scbc, ok := err.(github.StateCannotBeChanged); ok {
 				resp := fmt.Sprintf("failed to re-open PR: %v", scbc)
 				return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
 			}
+		} else {
+			// Add a comment after reopening the PR to leave an audit trail of who
+			// asked to reopen it.
+			response := fmt.Sprintf("Reopened this PR.")
+			if err := gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, commentAuthor, response)); err != nil {
+				log.WithError(err).Errorf("Failed adding comment while reopening the PR")
+			}
 		}
 		return err
 	}
 
-	response := fmt.Sprintf("Reopening this issue.")
-	if err := gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, commentAuthor, response)); err != nil {
-		log.WithError(err).Errorf("Failed adding comment while reopening the issue")
-	}
-
 	log.Infof("/reopen issue")
 	err = gc.ReopenIssue(org, repo, number)
 	if err != nil {
 		if scbc, ok := err.(github.StateCannotBeChanged); ok {
 			resp := fmt.Sprintf("failed to re-open Issue: %v", scbc)
 			return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
 		}
+	} else {
+		// Add a comment after reopening the issue to leave an audit trail of who
+		// asked to reopen it.
+		response := fmt.Sprintf("Reopened this issue.")
```

Comment:
We should probably do this for `close` as well:

https://github.com/kubernetes/test-infra/blob/3a1230534d81f4a193f2ceb14bdef273a39fe141/prow/plugins/lifecycle/close.go#L90